### PR TITLE
Add recipe for python-telegram-bot

### PR DESCRIPTION
**Pull request by mdraw**

### CODE REVIEW

File: recipes/telegram-bot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+test:
+  imports:
+    - telegram
+    - telegram.base
+    - telegram.ext
+
+about:
+  home: https://github.com/python-telegram-bot/python-telegram-bot
+  license: GPL-3.0
```
Comment (synapticarbors) [inline recipes/telegram-bot/meta.yaml]: From all of the meta-data and READMEs, it looks like the project switched to using LGPL:

https://pypi.org/project/python-telegram-bot/#license

If you agree, than please also change the `license_file` to `LICENSE.lesser`

**Comment by mdraw** [comment]
The CI build output of the failing runs suggest that there is a temporary issue with PyPI, not with the recipe. Is there a way to re-run CI on the newest revision?

**Comment by synapticarbors** [comment]
@mdraw I closed and re-opened the PR to restart the CI, hopefully the intermittent issues that were causing problems before will be gone.

**Comment by mdraw** [comment]
Travis works now after another manual restart (figured out I can restart it myself by logging into Travis with my GitHub account). Is the recipe ready to be merged?

**Comment by mdraw** [comment]
Thanks @synapticarbors!

**Pull request merged**